Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM alpine:latest
 RUN apk add --no-cache ca-certificates && update-ca-certificates
 ADD https://get.aquasec.com/microscanner /
+RUN microscannerHash=`md5sum microscanner | awk '{ print $1 }'`; \
+    if [ "$microscannerHash" != "1d911d709cb9efb02a4661926af979a6" ]; then echo Microscanner md5 is incorrect;exit 1; fi
```

Comment:
@oranmoshai shouldn't this be using a stronger hashing algorithm like sha256 or sha512?

This operation can be optimised using a single RUN step:

```Dockerfile
FROM alpine:latest
ARG token
RUN apk add --no-cache ca-certificates \
  && update-ca-certificates \
  && wget https://get.aquasec.com/microscanner -O /microscanner \
  && echo "72fd95ef5d343915c37ad487ba83da56e4d79d2f999cbdb2bfb1afda0d6bd7bb  /microscanner" | sha256sum -c - \
  && chmod +x /microscanner \
  && /microscanner ${token}
```

Reply:
Agreed, this would be better